src/main/java/uk/gov/companieshouse/acsp/models/filing/ACSP.java

```diff
@@ -10,8 +10,8 @@ public final class ACSP {
     @JsonProperty("payment_method")
     private String paymentMethod;
 
-    @JsonProperty("correspondence_address")
-    private Address correspondenceAddress;
+    @JsonProperty("service_address")
+    private ServiceAddress serviceAddress;
     @JsonProperty("office_address")
```

Comment:
this needs to be "registered_office_address"

Reply:
done